docs/fundamentals/dataclass/access.md

```diff
@@ -1,87 +1,155 @@
 # Access Modality
 
-```{tip}
-It is strongly recommended to go through the {ref}`access-elements` section first before continuing.
+After {ref}`constructing a multi-modal Document or DocumentArray <mm-construct>`, you can directly access your custom-defined
+modalities by their names.
+
+```{admonition} Return types
+:class: seealso
+
+Accessing a modality always returns a Document or a DocumentArray, instead of directly returning the data stored in them.
+
+If you are interested in the principles behind this design decision, you can learn more [here](TODO LINK TO BLOG POST).
```

Comment:
```suggestion
Accessing a modality always returns a Document or a DocumentArray, instead of directly returning the data stored in them.

This allows you, for example, to store embeddings directly in this Document.
```